conf/application.conf

```diff
@@ -170,6 +170,11 @@ microservice {
       port = 9730
     }
 
+    bars {
+      host = localhost
+      #9078 points to direct-debit-stubs, can't point to bars-stubs because they haven't implemented that endpoint ...
```

Comment:
Nb: May need to add DIRECT_DEBIT_STUBS to the SSTTP profile

Reply:
Yes, I'll add it.